src/test/java/com/google/sps/servlets/ScheduleServletTest.java

```diff
@@ -1,95 +0,0 @@
-// Copyright 2019 Google LLC
```

Comment:
Couldn't you refactor this a bit to instead test the new `getSelectedTasks` method? The idea would be quite similar just that the source of truth for which tasks exist is dependent on the return value of `tasksClientAdapter.getTask(tasksListId, id)` which you could mock.